weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1801,6 +1801,169 @@ def scorer_delete_v2(self, req: tsi.ScorerDeleteV2Req) -> tsi.ScorerDeleteV2Res:
         result = self.obj_delete(obj_delete_req)
         return tsi.ScorerDeleteV2Res(num_deleted=result.num_deleted)
 
+    def evaluation_create_v2(
+        self, req: tsi.EvaluationCreateV2Req
+    ) -> tsi.EvaluationCreateV2Res:
+        """Create an evaluation object.
+
+        Creates placeholder ops for evaluate, predict_and_score, and summarize methods.
+        """
+        # Create a safe ID for the evaluation
+        safe_name = object_creation_utils.make_safe_name(req.name)
+        evaluation_id = f"evaluation_{safe_name}"
+
+        # Create placeholder evaluate op
+        evaluate_op_req = tsi.OpCreateV2Req(
+            project_id=req.project_id,
+            name=f"{evaluation_id}_evaluate",
```

Comment:
Isn't it supposed to be `MyEvaluation.evaluate`?

Reply:
fixed